src/styles/header.scss

```diff
@@ -0,0 +1,54 @@
+@import "../utils/variables.scss";
+@import "../utils/mixins.scss";
+
+.header {
+  @include flex-space-between;
+  align-items: center;
+  text-transform: uppercase;
+  padding: 0 50px;
+  // box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-img {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  color: $nav-link-color;
+  text-decoration: none;
+
+  &:hover {
```

Comment:
Add a transition to the hover effect to make the color change smooth. For example, you can add 'transition: color 0.3s;' to the '.nav__link' class.